### ISSUE TITLE
Add accountId in createAccount

### ISSUE THREAD

**Issue by ChihChengLiang** [thehubbleproject/RedditHubble#58]
Thanks to @kilic for reasoning this.

We should add accountID in CreateAccount. The `toIndex` says which position in the state tree that we are creating an account. And the accountID says which position in the public key tree that has the public key to the owner of the account.

```
struct CreateAccount {
        uint256 toIndex;
->    uint256 accountID;   <-
        uint256 tokenType;
        bytes signature;
    }
```

**Comment by vaibhavchellani** [comment]
I think its actually toIndex and stateID

**Comment by ChihChengLiang** [comment]
Fixed in #96, we have both accountID and stateID
